src/components/CardPhone/CardPhone.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import './cardphone.scss';
+
+export const CardPhone: React.FC = () => {
+  return (
+    <article className="card">
+      <img
+        src="../img/phones/apple-iphone-14-pro/spaceblack/00.webp"
+        alt="Iphone 14 Midnight"
+        className="card__image"
+      />
+
+      <h2 className="card__title">
+        Apple iPhone 14 Pro 128GB Deep Purple (MQ0G3)
+      </h2>
+
+      <p className="card__price-value">$999</p>
+
+      <div className="card__params">
+        <div className="card__param-pair">
+          <p className="card__params-info">Screen</p>
+          <p className="card__params-value">6.1” OLED</p>
+        </div>
```

Comment:
```suggestion
        <p className="card__param-pair">
          <span className="card__params-info">Screen</span>
          <span className="card__params-value">6.1” OLED</span>
        </p>
```
put <p> when it's completed thought

Reply:
change everywhere